website/docs/r/azure_secret_backend_role.html.md

```diff
@@ -64,7 +64,9 @@ The following arguments are supported:
 * `azure_groups` - List of Azure groups to be assigned to the generated service principal.
 * `azure_roles` - List of Azure roles to be assigned to the generated service principal.
 * `application_object_id` - Application Object ID for an existing service principal that will
-   be used instead of creating dynamic service principals. If present, `azure_roles` will be ignored.
+   be used instead of creating dynamic service principals. If present, `azure_roles` and `permanently_delete` will be ignored.
+* `permanently_delete` - Indicates whether the applications and service principals created by Vault will be permanently
```

Comment:
nit: since we're here, was wondering if we could also fix the small inconsistency in this file and add Optional / Required to all schema params respectively :)